third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -329,7 +329,7 @@ def to_gbq(
             >>> df = bpd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
             >>> destination = df.to_gbq(ordering_id="ordering_id")
             >>> # The table created can be read outside of the current session.
-            >>> bpd.close_session()  # For demonstration, only.
+            >>> bpd.close_session()
```

Comment:
Let's keep this comment or find a way to rephrase so it's 100% clear that the user doesn't have to close the session.

```suggestion
            >>> bpd.close_session()  # Optional, to demonstrate a new session.
```